graph_nets/blocks.py

```diff
@@ -437,7 +437,7 @@ def __init__(self,
     with self._enter_variable_scope():
       self._edge_model = edge_model_fn()
 
-  def _build(self, graph):
+  def _build(self, graph, **kwargs):
```

Comment:
Could you make the signature be?

`_build(self, graph, edge_model_kwargs=None)`

This may add a little more boilerplate when callign it, but is more explicit and should be easier to maintain in case we need to add another kwargs argument in the future. Also it is closer to the other models (e.g. GraphNetwork which has three of these).

(Here and in other blocks)